README.md

```
@@ -13,28 +13,45 @@
 import "github.com/go-coldbrew/errors"
 ```
 
-Package errors provides an implementation of golang error with stack strace information attached to it, the error objects created by this package are compatible with https://golang.org/pkg/errors/
+Package errors is a drop\-in replacement for the standard library "errors" package that adds stack trace capture, gRPC status codes, and error notification support.
 
-How To Use The simplest way to use this package is by calling one of the two functions
+All functions from the standard library errors package are re\-exported: [Is](<#Is>), [As](<#As>), [Unwrap](<#Unwrap>), [Join](<#Join>), and [ErrUnsupported](<#ErrUnsupported>). This allows you to use this package as your sole errors import:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)